bites_test.py

```diff
@@ -635,5 +652,229 @@ def test_play_full_game_calls_take_all_turns_first_and_then_print_scores(
       mock.call.printing_the_score()]
     self.assertEqual(manager.mock_calls, expected_calls)
 
+class RenderGameTest(unittest.TestCase):
+  def test_render_game_prints_player_name_and_hand_for_one_player(self):
+    # test 85
+    class FakePlayer():
+      def __init__(self, name):
+        self.name = name
+        self.hand = {}
```

Comment:
I haven't seen any tests to check that a non-empty hand is printed. I suggest that this scenario is covered too.